perf-tools/utils/log.go

```diff
@@ -19,18 +19,8 @@
 package utils
 
 import (
-	"go.uber.org/zap/zapcore"
-
 	"github.com/apache/yunikorn-core/pkg/log"
 	"go.uber.org/zap"
 )
 
-var Logger *zap.Logger
-
-func init() {
-	Logger = log.Logger()
-}
-
-func SetLogLevel(level int) {
-	log.InitAndSetLevel(zapcore.Level(level))
```

Comment:
Could we rewrite this function by `log.UpdateLoggingConfig`?

Reply:
Just moved it back. 

<img width="1103" alt="image" src="https://github.com/apache/yunikorn-release/assets/26764036/28ef466a-b69f-4258-84eb-72891b2f9abc">